dfds_ds_toolbox/analysis/plotting.py

```diff
@@ -144,45 +143,25 @@ def get_trend_stats(
 
 
 def plot_regression_predicted_vs_actual(
-    y_true: np.array, y_pred: np.array, title: str = "", extra_text: str = "", band_pct: float = 0.1
+    y_true: np.array, y_pred: np.array, alpha: float = 0.2
```

Comment:
This changes the public api, right? I suggest changing the PR title or description to reflect this change in the auto-generated release notes for v0.9.0.

Reply:
How should I indicate a breaking change in the title?
"BREAKING plot_regression_predicted_vs_actual - remove confidence band"?

Reply:
You can indicate it in the description too, I think. It's just because the release notes can be auto-generated based on the PRs title and description and if there are a lot of features then it becomes easier to highlight breaking changes when releasing.

Reply:
I have found a guide to autogenerating release notes and made a PR here. https://github.com/dfds-data/dfds-ds-toolbox/pull/15

Reply:
Cool!